### PR TITLE
Immediate Forward Plugin Configurable Channel 

### PR DESCRIPTION
**Pull request by jwillmartin**
modified:   src/v2i-hub/DsrcImmediateForwardPlugin/manifest.json
	modified:   src/v2i-hub/DsrcImmediateForwardPlugin/src/DsrcMessageManagerPlugin.cpp

# PR Details
## Description

- Made changes to Immediate Forward plugin manifest file, including channel number option in the default parameters for message_destination_1.
- Added a note to the manifest file, indicating users to change the channel to the one needed.
- Additionally, fixed typo in code, where the message frame included an unneeded space.

## Related Issue

https://github.com/usdot-fhwa-OPS/V2X-Hub/issues/427

## Motivation and Context

This change allows users to send messages to multiple RSUs transmitting on different channels

## How Has This Been Tested?

Tested locally and decoded messages forwarded from V2X Hub to the RSU, using wireshark to verify the changes. 
Uploaded to github and verified desired changes reflected in the automatically generated docker image.
Finally, tested functionality with both a DSRC RSU on channel 172 and a C-V2X RSU on channel 183. 

## Types of changes

- [x] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [x] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[V2XHUB Contributing Guide](https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/v2i-hub/DsrcImmediateForwardPlugin/manifest.json [hunk 1]
```diff
@@ -9,23 +9,23 @@
   "configuration": [
     {
       "key": "Messages_Destination_1",
-      "default": "{ \"Messages\": [ { \"TmxType\": \"SPAT-P\", \"SendType\": \"SPAT\", \"PSID\": \"0x8002\" }, { \"TmxType\": \"MAP-P\", \"SendType\": \"MAP\", \"PSID\": \"0x8002\" }, { \"TmxType\": \"PSM\", \"SendType\": \"PSM\", \"PSID\": \"0x8002\" } ,{ \"TmxType\": \"TMSG07\", \"SendType\": \"TMSG07\", \"PSID\": \"0x8002\" },{ \"TmxType\": \"TMSG03-P\", \"SendType\": \"TMSG03-P\", \"PSID\": \"0xBFEE\" }] }",
-      "description": "JSON data defining the message types and PSIDs for messages forwarded to the V2X radio at destination 1."
+      "default": "{ \"Messages\": [ { \"TmxType\": \"SPAT-P\", \"SendType\": \"SPAT\", \"PSID\": \"0x8002\", \"Channel\": \"172\" }, { \"TmxType\": \"MAP-P\", \"SendType\": \"MAP\", \"PSID\": \"0x8002\", \"Channel\": \"172\" }, { \"TmxType\": \"PSM\", \"SendType\": \"PSM\", \"PSID\": \"0x8002\", \"Channel\": \"172\" } ,{ \"TmxType\": \"TMSG07\", \"SendType\": \"TMSG07\", \"PSID\": \"0x8002\", \"Channel\": \"172\" },{ \"TmxType\": \"TMSG03-P\", \"SendType\": \"TMSG03-P\", \"PSID\": \"0xBFEE\", \"Channel\": \"172\" }] }",
```
Comment (paulbourelly999) [inline src/v2i-hub/DsrcImmediateForwardPlugin/manifest.json:12]: If this configuration overrides channel numbers hardcoded in other plugins maybe we should consider removing hardcoded values from everywhere and essentially only using this configuration parameter. We could still have a default but having an invisible default value, that is set in a per plugin basis seems hard to follow

Comment (jwillmartin) [reply src/v2i-hub/DsrcImmediateForwardPlugin/manifest.json:12]: @paulbourelly999 Changes have been made, tested, and pushed to this branch

**Review by paulbourelly999 — APPROVED** [review]
Looks good.
